content/docs/tasks/security/vault-ca/index.md

```diff
@@ -12,6 +12,18 @@ certificates in Istio.
 
 * Create a new Kubernetes cluster to run the example in this tutorial.
 
+## Certificate request flow
+
+At high level, an Istio proxy (i.e., Envoy) requests a certificate from Node Agent
+through SDS. Node Agent sends a CSR (Certificate Signing Request), with the Kubernetes service
+account token of the Istio proxy attached, to Vault CA. Vault CA authenticates and authorizes
+the CSR based on the Kubernetes service account token and returns the signed certificate
+to Node Agent, which returns the signed certificate to the Istio proxy.
+
+The administrators of Vault CA need to configure the authentication and authorization
```

Comment:
Change to:

You need to configure Vault CA to enable authentication and authorization of Kubernetes service accounts.

Reply:
Done. Thanks.